pkg/internal/checkup/executor/console/console.go

```diff
@@ -80,7 +80,7 @@ func (e Expecter) spawnConsole(timeout time.Duration) (*expect.GExpect, error) {
 		})
 	}()
 
-	e.opts = append(e.opts, expect.SendTimeout(timeout), expect.Verbose(true))
+	e.opts = append(e.opts, expect.SendTimeout(timeout), expect.Verbose(false))
```

Comment:
The flag enables log prints of the console conversation. See [here](https://github.com/google/goexpect/blob/ab937bf7fd6f93d0b1adaef7bf038f31f61bc96c/expect.go#L773C9-L773C16) and [here](https://github.com/google/goexpect/blob/ab937bf7fd6f93d0b1adaef7bf038f31f61bc96c/expect.go#L1237).

Reply:
See at the end of [this](https://github.com/kiagnose/kubevirt-dpdk-checkup/pull/207#issuecomment-1818935448) comment

Reply:
Commit message:
> Since the checkup already is logging the output it wants when the verbose flag is set

Do you mean the user-supplied `verbose` flag?

Reply:
our user supplied verbose flag is printing the desired portion of the console output (sometimes after some parsing & marshaling).
This flag `expect.Verbose(true)` however is printing the raw console batch struct, which is much more than we need, and is also redundant to what we print in our user supplied verbose flag.
By setting it to false I am doing a major cleanup to the logs, making it much clearer

Reply:
rephrased the commit message to make it clearer